### PR TITLE
Report errors caused by subprocesses

### PR DESCRIPTION
**Pull request by AlexBrownAMD**
And optionally print all subprocess output in verbose mode

### CODE REVIEW

File: Tensile/TensileCreateLibrary.py [hunk 1]
```diff
@@ -261,7 +276,12 @@ def buildSourceCodeObjectFile(CxxCompiler, outputPath, kernelFile):
               if globalParameters["PrintCodeCommands"]:
                 print(' '.join(bundlerArgs))
               # change to use  check_output to force windows cmd block util command finish
-              subprocess.check_output(bundlerArgs, stderr=subprocess.STDOUT)
+              try:
+                out = subprocess.check_output(bundlerArgs, stderr=subprocess.STDOUT)
```
Comment (nakajee) [inline Tensile/TensileCreateLibrary.py:280]: This part is already in try section.
Do we need to use nested try here?

Comment (AlexBrownAMD) [reply Tensile/TensileCreateLibrary.py]: Just posted an update to clean that up. Thanks!

Comment (nakajee) [reply Tensile/TensileCreateLibrary.py]: I found 1 more subprocess.check_output call at line 238 (before change) in TensileCreateLibrary.py.
Do we need to add print for the output of this call?

Comment (AlexBrownAMD) [reply Tensile/TensileCreateLibrary.py]: That one should also be covered by my most recent change

Comment (nakajee) [reply Tensile/TensileCreateLibrary.py]: I understand that the exception is captured.
I am just talking about print2 you added for all other check_output calls.
Like this.
  print2(out)

**Review by benjaminulmer — APPROVED** [review]
Still not perfect compared to the `check_call` we used to have, but it's an appreciated improvement.
All the appropriate calls have been updated as far as I can tell.

**Review by cgmillette — APPROVED** [review]
LGTM

**Pull request merged**